src/lang/locales/en-US.json

```diff
@@ -65,7 +65,8 @@
       "view-api": "View API collection",
       "view-symbol": "View symbol",
       "view-sample-code": "View sample code"
-    }
+    },
+    "view-more": "View More"
```

Comment:
Not a big deal, but "more" was not capitalized in the original string

Reply:
fixed https://github.com/apple/swift-docc-render/pull/747/commits/a1d78781cb39b92b452c05521ad197eff86d8e99